libraries/AP_AHRS/AP_AHRS_NavEKF.h

```diff
@@ -186,6 +186,9 @@ class AP_AHRS_NavEKF : public AP_AHRS_DCM {
     // Write position and quaternion data from an external navigation system
     void writeExtNavData(const Vector3f &pos, const Quaternion &quat, float posErr, float angErr, uint32_t timeStamp_ms, uint32_t resetTime_ms) override;
 
+    // Write velocity data from an external navigation system
+    void writeVisionSpeed(const Vector3f &vel, uint32_t timeStamp_ms) override;
```

Comment:
it's just a name but I would like to keep this similar to the "external nav" handling.  Basically the source is the same as "external nav" it's just that it's all velocity.  So I know the high level mavlink message is "VISION_SPEED" so in AP_VisualOdom the, "handle_vision_speed(..)" method is the correct name to use.  But once we're at the AHRS/EKF level I think we should make it more like "writeExtNavVelocityData(..)"... sorry to be picky but this applies to the variable names as well.

If this is too annoying a request I could take care of renaming the variables.

Reply:
OK, I will do it